data_prototype/wrappers.py

```diff
@@ -126,14 +127,16 @@ def _query_and_transform(self, renderer, *, xunits: List[str], yunits: List[str]
             return self._cache[cache_key]
         except KeyError:
             ...
-        # TODO decide if units go pre-nu or post-nu?
+
+        # Step 2
         for x_like in xunits:
             if x_like in data:
                 data[x_like] = ax.xaxis.convert_units(data[x_like])
         for y_like in yunits:
             if y_like in data:
                 data[y_like] = ax.xaxis.convert_units(data[y_like])
```

Comment:
```suggestion
                data[y_like] = ax.yaxis.convert_units(data[y_like])
```
?

(Will it be easy to do this for z as well?)

Reply:
yes, but leaving generalizing for later.